src/crimson/osd/pg.cc

```diff
@@ -847,22 +864,24 @@ std::optional<hobject_t> PG::resolve_oid(
   const SnapSet &ss,
   const hobject_t &oid)
 {
+  logger().debug("{} oid.snap={},head snapset.seq={}",
+                 __func__, oid.snap, ss.seq);
   if (oid.snap > ss.seq) {
+    // perfrom a read from head
```

Comment:
Either remove this comment or make it more explanatory:
```
    // Because oid.snap > ss.seq, we are trying to read from a snapshot
    // taken after the most recent write to this object.  Read from head.
```